core/src/main/java/io/questdb/std/str/DirectByteCharSink.java

```diff
@@ -33,7 +33,7 @@
 
 import java.io.Closeable;
 
-public class DirectByteCharSink implements Mutable, ByteSequence, Closeable {
+public class DirectByteCharSink extends AbstractCharSink implements Mutable, ByteSequence, Closeable {
```

Comment:
Is it  really needed ? 
I'm asking because AbstractCharSink comes with bit of a baggage (doubleDigits)  that looks weird compared to this class.

Reply:
I was asking myself the same question. In fact, I had a working version where I added a utility method `utf16ToUtf8()` to `Chars`. 

So why is it like this? 2 reasons:
1. I found a bit odd that a class `DirectByte*CharSink*` was not actually a `CharSink`. 
2. @bluestreak01 was against the utility method for transcoding.  He strongly advocated for re-using the `encodeUtf8()` methods from `CharSink 

The other option could be to create a brand new implementation of `CharSink`, but that did not feel good either. 

wdyt?

Reply:
I guess it'd be better to move doubleDigits to where they're really needed  but that's outside of PR scope .